samples/spring-java-fibonacci-action/src/main/java/com/example/KalixServer.java

```diff
@@ -0,0 +1,25 @@
+package com.example;
+
+import com.example.fibonacci.FibonacciAction;
+import kalix.javasdk.Kalix;
+import kalix.springsdk.action.ReflectiveActionProvider;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+
+// TODO: this should be part of the Spring SDK and automagically find the components to register
+@Component
+public class KalixServer {
```

Comment:
that won't stay. We will need something in the SDK that does the wiring as needed. I just plug it here like that because one to see a Spring Boot Kalix thing running :-)